doc/release/yarp_3_9_master/master.md

```diff
@@ -30,4 +30,9 @@ Added new device `LLM_nwc_yarp`.
 
 # yarp::sig::Sound
 
-`yarp::sig::sound` refactored to avoid the internal use (private implementation) of yarp::sig::Image data type.
\ No newline at end of file
+`yarp::sig::sound` refactored to avoid the internal use (private implementation) of yarp::sig::Image data type.
+
+# deprecation
+
+Removed deprecated devices: transformClient, transformServer, analogWrapper, controlBoardWrapper
```

Comment:
Can you mention here that the `analogServer` device is removed? As the name of the yarp device used in yarpdev/yarprobotinterface is analogServer, not `analogWrapper`.

Reply:
yes, in my mind they are the same thing since the folder/file name is analogwrapper but the device is called analogserver. An unfortunate and confusing mistake that was never fixed.